docs/devguide.md

```diff
@@ -31,11 +34,11 @@ layout:
     │   └── hack                # Non-build related scripts
     │   └── jenkins             # Jenkins configuration
     │   └── pkg                 # Contrib golang code
-    ├── charts                  # Helm charts for deployment
-    │   └── catalog             # Helm chart for deploying the catalog
-    │   └── ups-broker          # Helm chart for deploying the user-provided service broker
     ├── docs                    # Documentation
     ├── pkg                     # Contains all non-"main" Go packages
+    ├── plugin                  # Plugins for API server
+    ├── scripts                 # Push images to registry
+    ├── test                    # Test include integration, e2e
```

Comment:
The wording here is a bit strange to me. I'd recommend `# Integration and e2e tests`

Reply:
Thanks for review and suggestion. I will update it.